components/ui/themes/DefaultTheme.ts

```diff
@@ -1312,11 +1312,11 @@ export const DefaultTheme = {
 
   ICON_SMALL_SIZE: 16,
   ICON_MID_SIZE: 22,
+  ICON_LARGE_SIZE: 33,
   PinIcon: require('../../../assets/pin_icon.png'),
   CloseCard: require('../../../assets/card_bg.png'),
   CardBackground: require('../../../assets/card_bg.png'),
   OpenCard: require('../../../assets/card_bg.png'),
-  activationPending: require('../../../assets/pending_activation.png'),
```

Comment:
pending activation image is not removed from the assets folder.

Reply:
Done